typings/rendition.d.ts

```diff
@@ -466,7 +466,7 @@ declare module 'rendition' {
 		cellAttributes?:
 			| React.AnchorHTMLAttributes<HTMLAnchorElement>
 			| ((value: any, row: T) => React.AnchorHTMLAttributes<HTMLAnchorElement>);
-		field: keyof T;
+		field: string | number | undefined;
```

Comment:
Why is this a required change for node 6 to work?
I would prefer to avoid this if we can, since it would mean that passing a field name that's not part of the object will longer result a typing error.
If newer TS keyof semantics about Symbols is messing with it, you could use
```suggestion
		field: keyof T & string;
```